### DISCUSSION TITLE
Please don't use automatic plural in the breadcrump.

### DISCUSSION THREAD

**Discussion by dmAssecor** [statamic-rad-pack/runway#337] [Help] [answered]
Hey, your package works pretty well for us, but we have a problem in your breadcrumb. Our application is in German and when the breadcrumb is displayed, it always adds an "s" after the word. This is because you set every breadcrumb in plural in your ResourceController.php. This works for English words great, but not in other languages.

Is there any way to include an option to turn the plural off or assign a value to the breadcrumb yourself?

**Comment by duncanmcclean** [comment, marked as answer]
The latest release includes two new configuration options which allow you to override the `singular`/`plural` titles used throughout Runway: [v5.3.0](https://github.com/duncanmcclean/runway/releases/tag/v5.3.0).

**Reply by dmAssecor** [reply]
Thank you! It work's great 👍
